hw1/hw1.txt

```diff
@@ -0,0 +1,9 @@
+Change Directory: cd directoryname
+Create a Directory: mkdir newdirectoryname
+Rename a Directory: mv originalname newname
+List the contents of a directory: ls -al
+Move a file to a new location: mv originalpath/filename newpath/
+Copy a file to a new location: cp originalpath/filename newpath/
+Search for a string in a file: tail filename | grep -nw "targetstring"
```

Comment:
Any particular reason why you pipe the output of tail to grep instead of calling grep on the file directly such as:

grep -nw "targetstring" filename